snaps-boot/randy-hosts.yaml

```diff
@@ -0,0 +1,191 @@
+--- 
```

Comment:
I don't think that the file name randy-hosts.yaml is very descriptive.  The other config files are located in the conf...sub directory.